module3-crud-and-data-models/r3-model-view-controller/README.md

```diff
@@ -0,0 +1,94 @@
+# Model View Controller (MVC)
+
+## Introduction to MVC projects
+
+When beginning a new project (especially, when using express), there are several different methods that can be used to set up the overall file structure and flow. One of the most commonly used architectural patterns is called MVC. This acronym stands for "Model, View, Controller".
+This pattern is favored because of its alignment with the computer science design principle, [**separation of concerns**](https://en.wikipedia.org/wiki/Separation_of_concerns). By dividing up responsibilities within our file structure, for example, we can have our db connection work in one file and api routes in another file, ...etc.
```

Comment:
```suggestion
This pattern is favored because of its alignment with the computer science design principle, [**separation of concerns**](https://en.wikipedia.org/wiki/Separation_of_concerns). By dividing up responsibilities within our file structure, for example, we can have our database connection work in one file and API routes in another file, and so on. This makes our code better organized, which ultimately helps ease the developer workflow and prevent bugs.
```